src/status_im/contexts/wallet/events.cljs

```diff
@@ -439,3 +439,32 @@
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
                 :duration 10000}]]])})))
+
+(rf/reg-event-fx
+ :wallet/save-address
+ (fn [_ [{:keys [address name color-id on-success on-error chain-short-names ens is-test]
+         :or   {on-success        (fn [])
+                on-error          (fn [])
+                name              ""
+                ens               ""
+                is-test           false
```

Comment:
this wouldn't handle cases where any of these are explicitly passed as `nil`

Reply:
That seems to be the correct behaviour? If it's `nil` explicitly, that's what should be sent to RPC? 

I'm not able to see the problem here.

Reply:
i don't know the api so no idea if this rpc call will fail if some args are nil, but if the intention of the lines above is to have default values for the rpc args, then might make sense to apply the defaults if a value is nil as well e.g. `{:is-test (when something true) ...}` will be passed as nil to the rpc call and might lead to unexpected behavior

Reply:
The rpc method is rather forgiving. The default object is being sent just for safe keeping and documentation of what is needed.